docs/about-us.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _about-us:
+
+********
+About Us
+********
+
+``meson-python`` is a project that extends the Meson_ build system, making it
+possible to it for Python_ packaging. It is 100% open source and is available
```

Comment:
Please remove the link from Python, and every repeated link to `Meson_` further down. The overuse of links, especially when they are repeatedly pointing to the exact same URL, hinders readability.